python/pyspark/sql/functions.py

```diff
@@ -1427,6 +1427,19 @@ def to_utc_timestamp(timestamp, tz):
     return Column(sc._jvm.functions.to_utc_timestamp(_to_java_column(timestamp), tz))
 
 
+@since(3.1)
+def timestamp_seconds(col):
+    """
+    >>> from pyspark.sql.functions import timestamp_seconds
```

Comment:
Can you set the session timezone? It caused SPARK-32088

Reply:
Thanks HyukjinKwon, will fix soon